### ISSUE TITLE
Add New Statsforecast method MFLES

### ISSUE THREAD

**Issue by tblume1992** [autogluon/autogluon#4697]
## Description
Statsforecast added MFLES /AutoMFLES into their framework a few months back, would be great if that could be added to the timeseries module which already supports most of their methods!

## References
Documentation: https://nixtlaverse.nixtla.io/statsforecast/docs/models/mfles.html
Original Code: https://github.com/tblume1992/MFLES

Their implementation is a port of my original code so let me know if there are any specific questions!

**Comment by shchur** [comment]
Hi @tblume1992, thank you for the suggestion, this sounds like a good idea since we already depend on StatsForecast.

I had a quick look at the model but wasn't able to figure out the specific requirements that must be fulfilled for the model to produce predictions for arbitrary `quantile_levels` (list of floats 0 < q < 1). It seems that cross-validation is needed, but I wasn't sure to what extent.

For some more context: in AutoGluon we return quantile forecasts by default, and all existing models are able to fit + predict all quantiles given a historic time series with at least `prediction_length + 1` observations. Do you have any recommendations on how we can achieve that for MFLES?

**Comment by tblume1992** [comment]
@shchur yeah there is no direct way like for an ARIMA but you should be able to use the conformal intervals produced from Nixtla. 

I see there also is the IMAPA model in list of models supported which also doesn't have a direct way for quantiles (I think) so do y'all use conformal intervals for that as well? Whatever works for that should work with MFLES.

**Comment by shchur** [comment]
Makes sense, we can do that. I just wanted to check if there was no other option that I was missing (e.g. based on using the quantile regression mode of LightGBM).

**Comment by tblume1992** [comment]
No unfortunately not! Definitely let me know if there are any other questions or issues with the integration!
